corehq/apps/aggregate_ucrs/migrations/0003_auto_20180825_1107.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.14 on 2018-08-25 11:07
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('aggregate_ucrs', '0002_auto_20180824_1319'),
```

Comment:
could just update the previous migration rather than having another one

Reply:
updated. out of curiosity is the main motivation for doing something like this (test?) performance? or just cleaner housekeeping?

Reply:
mostly clean housekeeping though having tonnes of migrations does make deploy slower